### PR TITLE
Adds default sidekiq config; starts sidekiq if APP_ROLE == 'sidekiq'

### PR DESCRIPTION
**Pull request by whereismyjetpack**
Sets a "sane" default config for sidekiq and runs `/app/bin/sidekiq` instead of `rails s` if the containers APP_ROLE is set to "sidekiq"

Related: #217

### CODE REVIEW

File: start.sh [hunk 1]
```diff
@@ -1,12 +1,18 @@
...
 
-echo "starting rails"
-rm -f tmp/pids/server.pid
-bundle exec rails db:create
-bundle exec rails db:migrate
-bundle exec rails solr:init
-bundle exec rails s -b '0.0.0.0'
+if [ ${APP_ROLE:-app} == "sidekiq" ]; then
+    echo "starting sidekiq"
+    /app/bin/sidekiq
```
Comment (awead) [inline start.sh]: @whereismyjetpack I'm assuming this means that if there's a docker container (?) running with the APP_ROLE of sidekiq, then it just starts sidekiq?

File: start.sh [hunk 2]
```diff
@@ -1,12 +1,18 @@
-set -e 
+set -e
 
 if [ ${RAILS_ENV:-development} != "production" ]; then
   bundle check || bundle
+  yarn
```
Comment (awead) [inline start.sh]: @whereismyjetpack this doesn't seem related to sidekiq. Is it?

File: start.sh (hunk 1 above)
Comment (whereismyjetpack) [reply start.sh]: This is correct. the deployment for sidekiq sets that variable, and then runs as that "role" this allows both "roles" to share the same entrypoint to bootstrap the environment

File: start.sh (hunk 2 above)
Comment (whereismyjetpack) [reply start.sh]: removed.

Comment (awead) [reply start.sh]: @whereismyjetpack cool. Does it belong someplace else? I don't want to loose track of it if it is... we can ticket it or something if needed.

Comment (whereismyjetpack) [reply start.sh]: I'll submit it as another PR!
